sdk/search/search-documents/src/indexModels.ts

```diff
@@ -37,6 +37,26 @@ export interface SearchIndexingBufferedSenderOptions {
    * Indicates if autoFlush is enabled.
    */
   autoFlush?: boolean;
+  /**
+   * Initial Batch Document Count
+   */
+  initialBatchDocumentCount?: number;
+  /**
+   * Flush Window
+   */
+  flushWindowInMs?: number;
+  /**
+   * Maximum number of Retries
+   */
+  maxRetries?: number;
```

Comment:
Somehow in these options we need to document that we're doing a linear backoff based on the number of retries, since I think customers might expect an exponential backoff instead if we don't specify

Reply:
But, it is exponential back off. Am I wrong? In every retry, the delay between the retries in increased (until it reaches the maxRetryDelayInMs)

Reply:
Linear back-off increases the timeout by a fixed amount after each attempt(delay * attempt).  

Exponential back-off doubles the timeout after each attempt (2^attempt * delay).

Reply:
Thanks for explaining that. I have changed the code to include this logic